keras_retinanet/utils/image.py

```diff
@@ -62,14 +78,42 @@ def preprocess_image(x, mode='caffe'):
     """
     # mostly identical to "https://github.com/keras-team/keras-applications/blob/master/keras_applications/imagenet_utils.py"
     # except for converting RGB -> BGR since we assume BGR already
+
+    band_values = ['B', 'G', 'R', 'N']
+    imagenet_values = [103.939, 116.779, 123.68]
+
     x = x.astype(keras.backend.floatx())
     if mode == 'tf':
-        x /= 127.5
+        if bit_depth == '8':
+            x /= 127.5
+        elif bit_depth == '16':
+            x /= 32767.5
         x -= 1.
+
     elif mode == 'caffe':
-        x[..., 0] -= 103.939
-        x[..., 1] -= 116.779
-        x[..., 2] -= 123.68
+        if statistics:
+            if preprocess == 'mean':
+                for i, band in enumerate(band_values[:channels]):
+                    x[..., i] -= statistics['mean'][bit_depth][band]
+            elif preprocess == 'mean_std':
```

Comment:
What about to use just **std** instead of **mean_std**? Do we hae any constraints with it?

Reply:
Do you mean just rename mean_std to std? If yes, how about centering and standardization or smth like this. Or you want to add one else type of preprocessing by std?

Reply:
Yes, I mean just to rename these words. I agree with you, **centering** and **standardization** would be really great names!